src/Models/User.php

```diff
@@ -144,6 +144,8 @@ class User extends Model implements AuthenticatableContract, CanResetPasswordCon
      */
     protected $hidden = ['password', 'remember_token'];
 
+    protected $appends = ['email'];
```

Comment:
Can you add the missing documentation block here ?

Reply:
done